jlm/llvm/opt/alias-analyses/PointerObjectSet.hpp

```diff
@@ -787,11 +832,33 @@ class PointerObjectConstraintSet final
   Clone() const;
 
 private:
+  /**
+   * Creates the special subset graph containing both regular nodes and dereference nodes,
+   * used by DoOfflineVariableSubstitution().
+   *
+   * Some nodes are marked as direct, when all subset predecessors are known offline.
+   * They can:
+   *  - only be n(v) nodes, where v is a register
+   *  - not be the return value of a function call
+   *  - not be an argument of a function body
```

Comment:
Right, here is the definition of it. Maybe add some breadcrumbs in the other comments?